pkgs/by-name/st/STC/package.nix

```
@@ -0,0 +1,54 @@
+{ lib, stdenv, fetchFromGitHub, writeText }:
+
+stdenv.mkDerivation rec {
+  pname = "STC";
```

*The pname and the version attribute must not contain uppercase letters*
> https://github.com/NixOS/nixpkgs/tree/master/pkgs/README.md

```suggestion
  pname = "stc";
```